plugins/catalog-graph/src/components/EntityRelationsGraph/EntityRelationsGraph.tsx

```diff
@@ -91,6 +93,8 @@ export const EntityRelationsGraph = ({
   relationPairs?: RelationPairs;
   className?: string;
   zoom?: 'enabled' | 'disabled' | 'enable-on-click';
+  renderNode?: any;
+  renderLabel?: any;
```

Comment:
Never use any in public interfaces, and preferably never internally either. :)

Seems to me that this would more complex than the current pull request suggests. This would require breaking out their (pretty complex) prop types and making those public and exported as well.

Reply:
Hi @freben , I have updated the prop types with proper format ( same as existing custom node and custom label implementation). Also added test cases for new features. Please review.